CHANGES.rst

```diff
@@ -1,6 +1,10 @@
 1.10.3 (unreleased)
 ===================
 
+cube_build
+----------
+- Wavelength range of NIRSpec IFU cubes are set by cubepars reference file #7559
```

Comment:
PR number needs to be enclosed in square brackets at the end of the line. Can't be properly parsed this way.

Reply:
Oh, and regarding the statement itself, I thought the description of the PR said that the wavelength range is not yet specified in the cubepars ref file, so in the meantime this PR will use the wavelength range from the msm/emsm mode. This says it's coming from cubepars. And the sentence is just a statement of fact: it doesn't have a verb that says what's been changed.

Reply:
Fixed log entry. Updated description of PR. NIRSpec has already provided me with an updated reference file. Waiting until the jwst pipeline can use stadatamodels >= 1.4.1

DO NOT MERGE PR UNTIL NEW REF FILE IS IN CRDS AND JWST pipeline can use stdatamodels >= 1.4.1